db/migrate/20180524144325_change_to_github_profile.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeToGithubProfile < ActiveRecord::Migration[5.2]
+  def change
+    User.update_all("github = trim(leading 'http(s|)://(www|)github.com/' from github)")
```

Comment:
Let's make it in data_migration instead of schema migrations.
We have this gem https://github.com/ilyakatz/data-migrate in project, so use it.

Reply:
Also add please add github to the seeds for users https://github.com/howtohireme/give-me-poc/blob/e27a3b4ce73a277b6420d4313570307e3f0fffbe/db/seeds.rb#L14